main.py

```diff
@@ -19,6 +20,21 @@ def initStartProgram():
     for _ in range(9):
         maker.append(datetime.datetime.today())
     
+    if(os.path.exists('time.json')) :
```

Comment:
```suggestion
    if (os.path.exists('time.json')):
```

컨벤션 고치기가 쉽지 않아보이네요.. ㅎㅎ